incubator/neo4j/templates/loadbalancer.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ template "fullname" . }}-loadbalancer"
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "name" . }}
+    component: "{{ .Values.loadBalancerComponent }}"
+spec:
+  type: {{ .Values.serviceType }}
+  ports:
+    - port: {{ .Values.httpPort }}
+      targetPort: {{ .Values.httpPort }}
+      name: browser
+    - port: {{ .Values.boltPort }}
+      targetPort: {{ .Values.boltPort }}
+      name: bolt
+  selector:
+    app: {{ template "name" . }}
```

Comment:
You should not make the service a component of its own but rather use the one it belongs to.

The selector needs a `component` label.